File: PYTHON/desafio2_verificarPrimo.py
def verificarPrimo(numero):
    if numero < 2:
        return False
    for i in range(2, numero-1):
        if(numero % i == 0):
            return False
    return True

def main():
    numero = 7
    print(verificarPrimo(numero))

if __name__ == "__main__":
    main()

File: JAVASCRIPT/parte-01/desafio2_verificarPrimo.js
function main() {
  var numero = 10;
  console.log(verificarPrimo(numero));
}

function verificarPrimo(numero) {
  if (numero < 2) {
    return false;
  }

  for (let i = 2; i < numero; i++) {
    if (numero % i == 0) {
      return false;
    }
  }
  return true;
}

main();
